### PR TITLE
Throw when used with wrong Polymer version

### PR DESCRIPTION
**Pull request by platosha**
Connected to vaadin/components-team-tasks#251

This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-element-skeleton/36)

### CODE REVIEW

**Comment by limonte** [comment]
Maybe we could add a little bit more information here. Instead of `'Wrong Polymer version'` I would like to see `'The current Polymer version is ' + Polymer.version`

File: vaadin-element.html [hunk 1]
```diff
@@ -29,6 +29,10 @@
   </template>
 
   <script>
+    if (!Polymer.Element) {
+      throw 'Wrong Polymer version, expected Polymer v2.0.0 or later.';
```
Comment (limonte) [inline vaadin-element.html]: `throw` should throw an `Error` object, not string. http://eslint.org/docs/rules/no-throw-literal

Comment (limonte) [inline vaadin-element.html]: not relevant anymore :)

**Comment by platosha** [comment]
@limonte done

**Comment by Saulis** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/36#-:-KktthIISlBR9-Z2d0Ft:bnfp4nl)*

**Comment by tomivirkki** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-element-skeleton/36#-:-KkuPz9toxgYPqhO6uN1:bnfp4nl)*

**Pull request merged**